### PR TITLE
Add unit tests for executed_ops & executed_denunciations

### PR DESCRIPTION
**Pull request by sydhds**
* [ ] document all added functions
* [ ] try in sandbox /simulation/labnet
  * [ ] if part of node-launch, checked using the `resync_check` flag
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

### CODE REVIEW

**Review by Eitu33 — APPROVED** [review]
Approving but I believe these tests could use step comments (you don't have to add them on the test you moved) as tests can teach workflow behaviour to newcomers 👍

File: massa-executed-ops/src/executed_ops.rs [hunk 1]
```diff
@@ -508,3 +387,184 @@ impl Deserializer<BTreeMap<Slot, PreHashSet<OperationId>>> for ExecutedOpsDeseri
         .parse(buffer)
     }
 }
+*/
```
Comment (Leo-Besancon) [inline massa-executed-ops/src/executed_ops.rs:390]: Why are these commented? Remove if its deadcode?

**Review by Leo-Besancon — APPROVED** [review]
Approving for the tests, just check the commented out section ;)

**Review by Leo-Besancon — APPROVED** [review]
Approving for the tests, just check the commented out section ;)

File: massa-executed-ops/src/executed_ops.rs (hunk 1 above)
Comment (sydhds) [reply massa-executed-ops/src/executed_ops.rs]: Removed

**Review by Leo-Besancon — DISMISSED** [review]

**Comment by sydhds** [comment]
bors merge

**Review by Leo-Besancon — APPROVED** [review]

**Comment by sydhds** [comment]
bors merge

**Pull request merged**